### DISCUSSION TITLE
Call history / Date and time together directly in the list

### DISCUSSION THREAD

**Discussion by charles-at-inux** [FossifyOrg/General-Discussion#537] [Ideas]
Hi guys,

in [Settings] menu, in [General] section there is an option to [Change date and time], may I kindly ask to Add boolean option bellow this - to enable function: "ALWAYS show FULL DATE and TIME directly in the Call history" list?

Looking forward to be able to configure this schema/format directly in Call history page:
[ +phone_number/contact string ]
[ sim# ] [ icon answd/missd ] YYYY-MM-DD HH:MM

PS: I would like to join the Fossify project and put the hands on.
As I was writing android apps 7y ago, i so need to refresh my skills first.

**Comment by naveensingh** [comment]
I think the current approach is fine. It's user-friendly and does not lead to visual clutter. The suggested design won't look okay on screens with small widths, we'll have to go multiple lines or enable marquee effect.

The 'Show call details' option still has both date and time. Is that not sufficient?

Also, when raising a feature request, please use the feature request option: https://github.com/FossifyOrg/Phone/issues/new/choose.

**Comment by charles-at-inux** [comment]
When using app on an standart phone screen as of today, the space to show YYYY-MM-DD HH:MM is there enough, even if large font size is selected.

Agree with You, that an small screen devices (like watches) it won't fit and probably breaks down somehow (I didn't test it).

The main point is, to keep it default as it is now, but to allow this option for advanced users by the option "Allways show Date and Time" (or something similar).

As I allready said, searching date in the Show call details is very impractical - aspecialy in the moment, when You wan't to just quickly scroll by eyes the Call history log, to quickly remind when it happend (the call/the missed call ..).

Feature request option url: OK, sure

**Reply by Aga-C** [reply]
Do you use 1.2.0 version of the app or older? Because in the latest version, call history has changed looks, and it's easier to tell the exact date than before.

**Reply by charles-at-inux** [reply]
Running  version 1.1.1 from F-Droid repo and I have read the changelog for 1.1.2 version also.
Is there any fossify "night builds" or "first testers" storage, where recent builds of apks can be grabbed from for testing?
Looking forward to help to the project anyway - testing new releases is probably good start?

**Reply by Aga-C** [reply]
It's not nightly. Check [Github Releases](https://github.com/FossifyOrg/Phone/releases).

If you want to help, and you don't feel ready for doing any development tasks, you can check the list here: https://github.com/orgs/FossifyOrg/discussions/69#discussioncomment-8009887.

**Reply by charles-at-inux** [reply]
Upgraded to v1.1.2  few moments ago, and I have to say, that I do understand @naveensingh reaction in the moment.
Call history looks easy to determinate the date in the moment.
THX!
